packages/api/src/index.ts

```diff
@@ -1,120 +1,6 @@
-import uma from "@uma/sdk";
```

Comment:
I'm assuming all this was just moved?

Reply:
Yes moved to the types file. Maybe it should be import type